### PR TITLE
Fixing Issue#363

### PR DESCRIPTION
**Pull request by nairashwin952013**
Consistency for our usage of the constant ```MantaClient.SEPARATOR```. Refer [Issue#363](https://github.com/joyent/java-manta/issues/363)

### CODE REVIEW

File: java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java [hunk 1]
```diff
@@ -95,7 +96,7 @@ public void validateAllSupportedCiphers() throws IOException, DecoderException {
     /* Test Utility methods used in all unit-tests. */
 
     private void validateEncryptedFileWithHeaderValues(final SupportedCipherDetails cipherDetails) throws IOException, DecoderException {
-        final String cipherName = cipherDetails.getCipherId().replaceAll("/","-");
+        final String cipherName = cipherDetails.getCipherId().replaceAll(SEPARATOR,"-");
```
Comment (dekobon) [inline java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java]: Although the forward slash in this case is the same as `SEPARATOR`, they are being used differently. They are separating different content. I would undo this change.

File: java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java [hunk 2]
```diff
@@ -151,7 +152,7 @@ private void validateEncryptedFileWithHeaderValues(final SupportedCipherDetails
 
     private void populateHeaderValuesFromResources(final MantaHttpHeaders getResponseHttpHeaders,
                                                    final SupportedCipherDetails cipherDetails) throws IOException {
-        final String cipherName = cipherDetails.getCipherId().replaceAll("/","-");
+        final String cipherName = cipherDetails.getCipherId().replaceAll(SEPARATOR,"-");
```
Comment (dekobon) [inline java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java]: Same comment above. The cipher separator isn't the same as the directory separator.

File: java-manta-examples/src/main/java/ClientEncryptionServerMultipart.java [hunk 3]
```diff
@@ -52,7 +52,7 @@ public static void main(String... args) {
     }
 
     private static void multipartUpload(EncryptedServerSideMultipartManager multipart) {
-        String uploadObject = "/" + mantaUsername + "/stor/multipart";
+        String uploadObject = MantaClient.SEPARATOR + mantaUsername + "/stor/multipart";
```
Comment (dekobon) [inline java-manta-examples/src/main/java/ClientEncryptionServerMultipart.java]: We should probably use the separator for the slashes in the `/stor/multipart` string as well.

File: java-manta-examples/src/main/java/ServerMultipart.java [hunk 4]
```diff
@@ -45,7 +45,7 @@ public static void main(String... args) {
     }
 
     private static void multipartUpload(ServerSideMultipartManager multipart) {
-        String uploadObject = "/" + mantaUsername + "/stor/multipart";
+        String uploadObject = MantaClient.SEPARATOR + mantaUsername + "/stor/multipart";
```
Comment (dekobon) [inline java-manta-examples/src/main/java/ServerMultipart.java]: We should probably use the separator for the slashes in the /stor/multipart string as well.

File: java-manta-examples/src/main/java/SimpleClientEncryption.java [hunk 5]
```diff
@@ -43,7 +43,7 @@ public static void main(String... args) throws IOException {
                 .setEncryptionPrivateKeyBytes(Base64.getDecoder().decode("RkZGRkZGRkJEOTY3ODNDNkM5MUUyMjIyMTExMTIyMjI="));
 
         try (MantaClient client = new MantaClient(config)) {
-            String mantaPath = "/" + mantaUserName + "/stor/foo";
+            String mantaPath = MantaClient.SEPARATOR + mantaUserName + "/stor/foo";
```
Comment (dekobon) [inline java-manta-examples/src/main/java/SimpleClientEncryption.java]: We should probably use the separator for the slashes in the /stor/foo string as well.

File: java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java (hunk 2 above)
Comment (nairashwin952013) [reply java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java]: Noted.

File: java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java (hunk 1 above)
Comment (nairashwin952013) [reply java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java]: Noted.

File: java-manta-examples/src/main/java/SimpleClientEncryption.java (hunk 5 above)
Comment (nairashwin952013) [reply java-manta-examples/src/main/java/SimpleClientEncryption.java]: Done.

File: java-manta-examples/src/main/java/ServerMultipart.java (hunk 4 above)
Comment (nairashwin952013) [reply java-manta-examples/src/main/java/ServerMultipart.java]: Done.

File: java-manta-examples/src/main/java/ClientEncryptionServerMultipart.java (hunk 3 above)
Comment (nairashwin952013) [reply java-manta-examples/src/main/java/ClientEncryptionServerMultipart.java]: Done.
